frontend/src/App.tsx

```diff
@@ -0,0 +1,74 @@
+import i18n from "i18next";
```

Comment:
Formatting for the TS files is all over the place. I'll get a PR with a formatted that runs with git hooks up this weekend (hopefully), then it can be applied to this PR.